packages/client/src/workflow-client.ts

```diff
@@ -777,10 +777,19 @@ export class WorkflowClient extends BaseClient {
         },
       },
     };
-    let response: temporal.api.workflowservice.v1.UpdateWorkflowExecutionResponse;
 
+    // Repeatedly send UpdateWorkflowExecution until update is >= Accepted or >= requested stage (if
```

Comment:
```suggestion
    // Repeatedly send UpdateWorkflowExecution until update is >= Accepted or >= `waitForStage` (if
```

Currently reads a bit like "requested" is the name of a stage